tests/test_network.py

```diff
@@ -121,26 +121,35 @@ def test_get_network_element_ids(self):
 
     def test_buses(self):
         n = pp.network.create_eurostag_tutorial_example1_network()
-        buses = n.get_buses()
+        buses_bus_breaker_view = n.get_buses()
```

Comment:
this is not bus_breaker_view by default?

Reply:
yes by default it is not busbreakerview but I can change it